samples/gibbs_ensemble/gibbs.py

```diff
@@ -0,0 +1,139 @@
+import socket
+import struct
+import pickle
+import numpy as np
+
+
+class MessageId:
+    START = 0
+    END = 1
+    MOVE_PART = 2
+    MOVE_PART_REVERT = 21
+    CHANGE_VOLUME = 3
+    CHANGE_VOLUME_REVERT = 31
+    EXCHANGE_PART_ADD = 4
+    EXCHANGE_PART_ADD_REVERT = 41
+    EXCHANGE_PART_REMOVE = 5
+    EXCHANGE_PART_REMOVE_REVERT = 51
+    ENERGY = 6
+
+
+class Client():
+    '''
```

Comment:
i think we should consistently use ```"""```

Reply:
I don't really want to do such things manually. I personally use """, whereas the previous author used '''. I don't really find it important. If autopep8 can be convinced to do this automatically, I'm fine with adding that rule

Reply:
it just about consistency... just thought that python code in the samples is much more prominent to users than code in the python interface that is not read directly too often. Search and replace is also trivial in this case but if you think that code style is not that important in the samples you can leave it as is.

Reply:
> For consistency, always use """triple double quotes""" around docstrings

[pep8](https://www.python.org/dev/peps/pep-0257/#id15)